src/coreclr/vm/jitinterface.h

```diff
@@ -439,6 +438,23 @@ class CEEInfo : public ICorJitInfo
     static size_t findNameOfToken (Module* module, mdToken metaTOK,
                             _Out_writes_ (FQNameCapacity) char * szFQName, size_t FQNameCapacity);
 
+#ifdef HOST_WINDOWS
+    static uint32_t ThreadLocalOffset(void* p);
+
+    // MethodTable to `typeIndex` map. `typeIndex` is embedded in the code during codegen. During execution
+    // corresponding thread static data blocks are stored in `t_threadStaticBlocks` array at the `typeIndex`.
+    static TypeIDMap g_threadStaticBlockTypeIDMap;
```

Comment:
A more appropriate place for this table would be appdomain.hpp (e.g. next to the VSD type ID map).